UnofficialSteamAuthenticator/Models/User.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Windows.UI.Xaml.Media.Imaging;
+using SteamAuth;
+
+namespace UnofficialSteamAuthenticator.Models
+{
+    /// <summary>
+    /// Type for showing users in a list
+    /// </summary>
+    internal class User : ModelBase
+    {
+        public ulong SteamId { get; }
+        public string Title { get; set; }
+        public string Content { get; set; }
+        public BitmapImage Avatar { get; set; }
+        private readonly SteamGuardAccount _steamGuardAccount;
+
+        public User(ulong steamId, string title, string content, BitmapImage avatar)
+        {
+            SteamId = steamId;
+            _steamGuardAccount = Storage.GetSteamGuardAccount(steamId);
+
+            Title = title;
```

Comment:
Just as a side node... [Even VS 2015 suggests removing `this` by default ;)](http://abload.de/img/untitledpks5j.png)